src/Fusion/HTTP.elm

```diff
@@ -258,13 +364,7 @@ view model =
                 , label = Input.labelHidden "request headers input"
                 , spellcheck = False
                 }
-            , Input.multiline [ padding 5 ]
-                { onChange = RequestBodyChanged
-                , text = requestBodyString model.currentRequest
-                , placeholder = Just (Input.placeholder [] <| text "request body")
-                , label = Input.labelHidden "request body input"
-                , spellcheck = False
-                }
+            , bodyView model.currentRequest
```

Comment:
Minor but I like to extract functions below their first use, so reading a file top to bottom is like going deeper into the tree if that makes sense.

Reply:
Sounds like a good idea, I'll change that 👍